Bird-Modules/Sources/Authentication/Resources/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,18 @@
+/* 
+  Localizable.strings
+  
+
+  Created by Gabriel Ferreira de Carvalho on 04/12/22.
+  
+*/
+"hello" = "Hello!";
+"message_1" = "To use our online features we need you to sign in. you just got to press the Sign in with Apple button below.";
+"message_2" = "You will be able to download and upload your decks and more features are coming your way in the future.";
```

Comment:
"You will be able to download and upload your decks. More features are coming for you soon." talvez fique melhor